pkgs/applications/editors/quartus-prime/quartus.nix

```diff
@@ -34,18 +34,18 @@ let
   };
 
   version = "20.1.1.720";
-  homepage = "https://fpgasoftware.intel.com";
+  # note that download URL below is specific to the version!
 
-  require = {name, sha256}: requireFile {
+  download = {name, sha256}: fetchurl {
     inherit name sha256;
-    url = "${homepage}/${lib.versions.majorMinor version}/?edition=lite&platform=linux";
+    url = "https://downloads.intel.com/akdlm/software/acdsinst/20.1std.1/720/ib_installers/${name}";
```

Comment:
```suggestion
    url = "https://downloads.intel.com/akdlm/software/acdsinst/${lib.versions.majorMinor}std.${lib.versions.patch version}/${lib.elemAt (lib.splitVersion version) 4}/ib_installers/${name}";
```

Reply:
Not 100% sure the versions have this format consistently, but at least if it's not updated this will break obviously and we can re-evaluate.